src/video/VideoRow.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import * as VideoActions from '../_actions/VideoActions';
+
+
+@connect()
+export default class VideoRow extends React.Component {
+
+    onClick(){
+        this.props.dispatch(VideoActions.changeVideo(this.props.url));
```

Comment:
This will be handled in the parent, so this component is simpler.
Pass onSelect prop to this, and onClick={onSelect}
Then dispatch (or actually the raw action prop will be called from parent)